packages/editor/src/types/config.ts

```diff
@@ -13,7 +13,10 @@ export type FieldOption<TValue = PrimitiveValue> = {
   value: TValue;
 };
 
+type Subsection = 'General' | 'Styling' | 'Behaviour';
```

Comment:
I initially also had an attribute "Position" here so that you can clearly define at which position inside the subsection the field should appear. However, this is not necessary as the fields are arranged in the order of the component config field (see GIF)